styles/index.css

```diff
@@ -1,48 +1,43 @@
 * {
     box-sizing: border-box;
+    font-family: Helvetica, sans-serif;
+}
+
+.heading-brushes,
+.heading-frames, 
+.heading-paint {
```

Comment:
Heading has very similar style. If you would like to change the color of all them this way it would be easier rather than going though every single heading. What do you think?